apps/server/lib/plugins/jellyfish.ts

```diff
@@ -0,0 +1,180 @@
+import { PluginDefinition } from '@balena/jellyfish-worker';
+import type { OAuth2Client } from '@ory/hydra-client';
+import type { Kernel } from 'autumndb';
+import qs from 'qs';
+import { defaultEnvironment as environment } from '@balena/jellyfish-environment';
+import { hydraAdmin } from '../hydra-sdk';
+
+export const jellyfishPlugin = ({
+	kernel,
+}: {
+	kernel: Kernel;
+}): PluginDefinition => {
+	return {
+		slug: 'plugin-jellyfish',
```

Comment:
I really don't like the idea of (1) naming a plugin for Jellyfish `plugin-jellyfish` and (2) defining it inline in the server. This looks like it shoule belong in a `jellyfish-plugin-oauth` repo or something instead.

Reply:
As I commented below, I've opened up a `product-os/jellyfish-plugin-oauth` repo and started bringing over your code: https://github.com/product-os/jellyfish-plugin-oauth/pull/1